### ISSUE TITLE
Changes.md link broken in previous releases

### ISSUE THREAD

**Issue by ruthhoffmann** [gap-system/GapWWW#153]
The link to Changes.md in the pages of the older versions is returning a 404 from github. 
For example on 

https://www.gap-system.org/Releases/4.10.2.html 

Changes.md points to

https://github.com/gap-system/gap/blob/v4.10.2/CHANGES.md

rather than 

https://github.com/gap-system/gap/blob/v4.11.0/CHANGES.md

Note that a similar issue with the link dropping to the right part of Changes.md was raised in #69 .

**Comment by fingolfin** [comment]
Presumable the problem here is that a single template is used to generate all these pages; hence even old versions like 4.8.10 have a link to a CHANGES.md file that never existed in that version.

I suggest that we simply always link to https://github.com/gap-system/gap/blob/master/CHANGES.md

If we want to be fancy, we could also link to the specific version, but that'll require adding the relevant anchor to each of these releases, as they unfortunately don't follow a regular patten. 
So e.g. https://github.com/gap-system/gap/blob/master/CHANGES.md#gap-488-august-2017 could be used for 4.8.8

I guess we could adjust the CHANGES.md so that the version headlines don't contain the release date (which we could just move to the next line. So this:
```markdown
## GAP 4.8.8 (August 2017)
```
could become something like this:
```markdown
## GAP 4.8.8
Released August 2017.
```
or this
```markdown
## GAP 4.8.8
GAP 4.8.8 was released August 2017.
```
This change can be implemented using a simple regex.

Of course doing that, one notices that a couple versions are missing in the CHANGES.md; e.g. 4.8.9. @alex-konovalov perhaps those versions only updated packages, no GAP code? In that case, we could add simple entries which state just that.

**Comment by alex-konovalov** [comment]
Agree - we need just link it to https://github.com/gap-system/gap/blob/master/CHANGES.md then. As for those missing versions, these are indeed updates of packages only, with the same core GAP system.

**Comment by fingolfin** [comment]
This is actually resolved by PR #142 without a need to change our CHANGES.md file
